python/ray/autoscaler/azure/defaults.yaml

```diff
@@ -36,11 +36,10 @@ provider:
 # How Ray will authenticate with newly launched nodes.
```

Comment:
Could you also update `python/ray/autoscaler/azure/example-minimal.yaml` to remove `ssh_private_key` and `ssh_public_key` as well?